### PR TITLE
Remove timeslots

### PR DESCRIPTION
**Pull request by claudioclutter**
We stopped using timeslots a while ago, so we might as well clean up the code (-80 lines) and make it more readable.

Without timeslot, a build has at most one deploy:
- External builds have exactly one deploy, which will happen at `deploy_at`.
- Internal builds have exactly zero deploys. 

Existing builds with multiple deploys will only show the first one in the "Builds" page.

---

After this PR, the builds page will look like this:

Screen Shot 2019-10-28 at 12 37 15 PM

At this point, we can clean up this page even more, by having each build+deploy in a single line.
We would have to change the HTML, adapt the CSS, and eventually bump up the versions of our CSS frameworks, so I would leave that job for a successive PR.

---

@erikstrottmann Thoughts? Suggestions? Feedback from QA? Thanks.

### CODE REVIEW

File: app/models/build.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Build < ActiveRecord::Base
   has_one_attached :manifest
   validates :package, presence: true
 
-  has_many :deploys, dependent: :destroy
+  has_one :deploy, dependent: :destroy
```
Comment (erikstrottmann) [inline app/models/build.rb]: What does this do for existing build records that have multiple deploys? Will `Build#deploy` return the first record it finds?

Comment (claudioclutter) [reply app/models/build.rb]: Exactly. I left that comment on the PR description

> Existing builds with multiple deploys will only show the first one in the "Builds" page.

**Comment by claudioclutter** [comment]
@erikstrottmann 

For default `deploy_time` sounds good… you can make a separate PR for that?

For the rest of the PR… do you think this will work better? Did you try locally /  on the review app? Everything 👍 ?

**Comment by erikstrottmann** [comment]
I’ll make a separate PR to update the default `deploy_time`.

I think this will work better. I tried on the review app and was able to upload a build, but it wasn’t automatically enqueued so I ran `heroku run rake deploys:enqueue -a mdma-field-pr-163`. The deploy failed (possibly because I uploaded a build of Dock instead of Ugly Sweater), and I’m not sure how to check the error message.

**Pull request merged**